README.md

```diff
@@ -487,3 +487,15 @@ optional arguments:
 
 - [ ] Errors are handled according to Pulp conventions
 - [ ] Docs for plugin are available (any location and format preferred and provided by plugin writer)
+
+
+# Contributing
+
+### Pull Request Checklist
+
+1. Unless a change is small or doesn't affect users, create an issue on
+https://pulp.plan.io/projects/pulp . Set the Category to "Installer".
```

Comment:
s/affect uesrs/affect plugin writers/
s/Set the Category to "Installer"//Add the tag "Plugin Template"./

Also remove the unnecessary space before the period.